soc/arm/nuvoton_npcx/common/scfg.c

```diff
@@ -64,6 +67,17 @@ static void npcx_pinctrl_alt_sel(const struct npcx_alt *alt, int alt_func)
 	}
 }
 
+static void npcx_pinctrl_psl_detect_mode_sel(uint32_t offset, bool edge_mode)
+{
+	struct glue_reg *const inst_glue = HAL_GLUE_INST();
+
+	if (edge_mode) {
+		inst_glue->PSL_CTS |= BIT(offset + 4);
```

Comment:
Can you create a macros to convert the PSL number to the edge and event bit position?  Something like

/* PSL input detection type is configured by bits 7:4 */
#define PSL_EDGE_BIT(bit) (bit + 4)
/* PSL input level are reported by bits 3:0 */
#define PSL_EVENT_BIT(bit) (bit + 0)

This will make the code easier to read.

Reply:
Done.